openshift/datagrid-service/prometheus-datagrid/3-install-datagrid.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+echo "Installing RHDG cluster"
+
+oc create -f https://raw.githubusercontent.com/jboss-container-images/jboss-datagrid-7-openshift-image/datagrid73-dev/services/datagrid-service-template.yaml
+
+RHDG_USER=developer
+RHDG_PASSWORD=developer
+
+oc new-app datagrid-service \
+  -p APPLICATION_USER=developer \
+  -p APPLICATION_PASSWORD=developer \
+  -e AB_PROMETHEUS_ENABLE=true \
+  -p IMAGE=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/jboss-datagrid-7/datagrid73-openshift:jb-datagrid-7.3-openshift-dev-rhel-7-containers-candidate-46261-20181219020350
```

Comment:
@oraNod here are the new params

Reply:
The brew image link should not really be part of the tutorial

Reply:
IOW, leave it out